coder_sniffer/DrupalPractice/Project.php

```diff
@@ -94,6 +94,14 @@ public static function getInfoFile(File $phpcsFile)
                 $infoFiles = glob("$dir/*.info");
             }
 
+            // Filter out directories.
+            $infoFiles = array_filter(
+                $infoFiles,
+                function ($fileName) {
+                    return is_file($fileName);
+                }
+            );
```

Comment:
That could be done with one line.
`$infoFiles = array_filter($infoFiles, 'is_file');`